backend/middleware/errorHandler.js

```diff
@@ -2,12 +2,17 @@ const { serviceUnavailableMessage } = require('../common-messages')
 const { logError } = require('../logError')
 
 const errorHandler = (error, req, res, next) => {
+  if (error.response.status === 404) {
+    res.status(404)
```

Comment:
would be good to test this scenario too

Reply:
yes - will pull new error handling to separate PR and add in